flake.lock

```diff
@@ -2092,11 +2092,11 @@
         "prometheus-haskell": "prometheus-haskell"
       },
       "locked": {
-        "lastModified": 1687261453,
-        "narHash": "sha256-aBgTvFjIaKNHlDnCBKX1RUNnRRHWKT8zn0hzaiZ0XDU=",
+        "lastModified": 1687418441,
+        "narHash": "sha256-sPmjWi4CeQKvNOd9PvmxSK7D5bpPUhnmmLbK+0h+chk=",
         "owner": "nammayatri",
         "repo": "shared-kernel",
-        "rev": "7640c1bd8ea8b4b3a93c54fcc34b53a72ceb498e",
+        "rev": "00952972cadc4b13d9ed4cebdc8f837ffc54c33a",
         "type": "github"
       },
```

Comment:
where is the shared-kernel commit id updated?

Reply:
it is updated